tests/unit/auth/test_auth_service.py

```
@@ -354,6 +354,81 @@ async def test_check_auth_manual_bearer_extraction(self):
+    """
+
+    @staticmethod
+    def _format_record(record):
+        try:
+            return record.getMessage()
+        except Exception:
+            return str(record.msg)
+
+    @pytest.mark.asyncio
+    async def test_valid_token_path_never_logs_credential_bytes(self, caplog):
+        secret = "eyJhbGciOiJSUzI1NiIsImtpZCI6IkRO"  # 32-char JWT-shaped sentinel
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "secret"

(S105)